pyproject.toml

```
@@ -16,7 +16,8 @@ exclude = ["tests", "tests.*"]
 
 [tool.poetry.dependencies]
 python = "3.9.*"
-"discord.py" = {url = "https://github.com/Rapptz/discord.py/archive/45d498c1b76deaf3b394d17ccf56112fa691d160.zip"}
+# It is expected that bots that use bot-core will define a striciter version of disnake.
+disnake = ">=2.4"
```

We should cap the major release for API-incompatibility reasons:
```suggestion
disnake = ">=2.4, <3"
```

> Side note: Are we using all 2.4 features? Shouldn't we just be able to do `>=2, <3`?

---

Yea good point, updated.

---

this can be `^2` instead of `>=2, <3`